google-style.yml

```diff
@@ -37,6 +35,8 @@ Metrics/PerceivedComplexity:
   Max: 10
 Naming/VariableNumber:
   Enabled: false
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
I think this will be fine. There are other changes that 2.3 brings but we should be able to handle them. We could even just add this to the cloud rubocop config files and not make it part of the standard. Up to you.